hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/RatisPipelineProvider.java

```diff
@@ -155,12 +147,25 @@ public Pipeline create(ReplicationFactor factor) throws IOException {
 
     Pipeline pipeline = Pipeline.newBuilder()
         .setId(PipelineID.randomId())
-        .setState(PipelineState.OPEN)
+        .setState(PipelineState.ALLOCATED)
         .setType(ReplicationType.RATIS)
         .setFactor(factor)
         .setNodes(dns)
         .build();
-    initializePipeline(pipeline);
+
+    // Send command to datanode to create pipeline
+    final CreatePipelineCommand createCommand =
+        new CreatePipelineCommand(pipeline.getId(), pipeline.getType(),
+            factor, dns);
+
+    dns.stream().forEach(node -> {
```

Comment:
Just a thought, probably need @nandakumar131  to weigh in too. 
1. Since we are using the Ratis client in the DN's we might be able to get away with posting this command only to any one of the data nodes.
2. If we are posting this command to all DNs, we must be prepared to handle the fact that 2 out of 3 creates in the DN will fail. 
3. You current code is correct, since the createPipeline inside the DN -- handles the fact that createpipeline can fail. 

I am just flagging that information here, so that next code reviewer can also reflect upon this. No work or change needed.

Reply:
I see that we have done exactly the same as the current code; so no issues.